lib/identity_doc_auth/acuant/acuant_client.rb

```diff
@@ -15,8 +15,10 @@ def initialize(**config_keywords)
         @config = Config.new(**config_keywords)
       end
 
-      def create_document
-        Requests::CreateDocumentRequest.new(config: config).fetch
+      def create_document(cropping_mode:)
```

Comment:
can we add like a `# @see IdentityDocAuth::CroppingModes` or something? otherwise its hard for me to tell what type this should be

Reply:
> can we add like a `# @see IdentityDocAuth::CroppingModes` or something? otherwise its hard for me to tell what type this should be

Would a YARD type be better?

```
# @param cropping_mode [IdentityDocAuth::CroppingModes]
```

(I'm a YARD novice, so unsure if this would be valid, since it's a module of constants 🤷 )

Reply:
Well at the end of the day, the YARD stuff is mostly for ourselves. But you're right, I don't think that would be valid, since the literal type is a string, not an instance of a module.

Another option to save ourselves some potential heartache in the future, we could do a manual type check inside the method

1. if we added an `ALL` constant on CroppingModes that's an array or set
2. something like:

```ruby
if !CroppingModes::ALL.include?(cropping_mode)`
  raise "unknown cropping_mode=#{cropping_mode}"
end
```

We could also reflect + iterate.. but personally I'd rather have the array

```
if CroppingModes.constants.none? { |name| CroppingModes.const_get(name) == cropping_mode }
  raise "unknown cropping_mode=#{cropping_mode}"
end
```

Reply:
Works for me :+1: Added the comment pointer in e84f87e and validation in e6f0b75.